aws/v19.0.0/release.diff

```diff
@@ -0,0 +1,156 @@
+# Generated with:                                                  # Generated with:
+# devctl release create --name 19.0.0-beta1 --base 19.0.0-alpha |  # devctl release create --name 19.0.0 --base 19.0.0-beta1 --pro
+apiVersion: release.giantswarm.io/v1alpha1                         apiVersion: release.giantswarm.io/v1alpha1
+kind: Release                                                      kind: Release
+metadata:                                                          metadata:
+  annotations:                                                       annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp        giantswarm.io/docs: https://docs.giantswarm.io/reference/cp
+  creationTimestamp: null                                            creationTimestamp: null
+  name: v19.0.0-beta1                                           |    name: v19.0.0
+spec:                                                              spec:
+  apps:                                                              apps:
+  - componentVersion: 1.24.1                                         - componentVersion: 1.24.1
+    name: aws-cloud-controller-manager                                 name: aws-cloud-controller-manager
+    version: 1.24.1-gs1                                         |      version: 1.24.1-gs7
+  - componentVersion: 1.15.0                                    |    - name: aws-ebs-csi-driver
```

Comment:
Why component version disappear in new one?

Reply:
typo/mistake lemme fix it

Reply:
fixed